prepare.sh

```diff
@@ -37,4 +37,4 @@ cat ~/.sbuildrc
 echo "Checkout workspace"
 
 mkdir src
-vcs import --recursive --input  "$REPOS_FILE" src
+vcs import --recursive --shallow --input  "$REPOS_FILE" src
```

Comment:
I think this would break the sha1 in the version number.

Reply:
Not the sha1, but the [tag-based version scheme](https://github.com/jspricke/ros-deb-builder-action/blob/52ed9a9257ca7597ce5e7bf87968b3f14e3e5901/build.sh#L71-L73), right?

Reply:
Might be worthwhile to test at some point but dropped for now